docs/overview/code-editor-environment.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Office Scripts Code Editor environment
 description: The prerequisites and environment information for Office Scripts in Excel on the web.
-ms.date: 05/27/2021
+ms.date: 11/04/2022
 ms.localizationpriority: medium
 ---
 
```

Comment:
Consider adding a note block after the first paragraph of the article to notify users that VS Code for Office Scripts is currently in preview and link to the VS Code article.

Reply:
I thought about it, but this article feels very introductory, whereas the VSCode feature is advanced. @jeremy-msft, thoughts?

Reply:
I was going to agree with @AlexJerabek until I saw this line in the article:

> The Code Editor is based on Visual Studio Code, so if you've used that environment before, you'll feel right at home.

Since we already make the comparison, the note block might not be too out of place.